pkg/build/admission/admission.go

```diff
@@ -35,7 +35,7 @@ type buildByStrategy struct {
 // on policy based on the build strategy type
 func NewBuildByStrategy(client client.Interface) admission.Interface {
 	return &buildByStrategy{
-		Handler: admission.NewHandler(admission.Create),
+		Handler: admission.NewHandler(admission.Create, admission.Update),
```

Comment:
I had discussed this with @bparees ... I'm fine with either way. Because it is a virtual resource, separating create from update didn't seem to gain us much. You still have separate permission for create or update of the actual resource and requiring that permissions be aligned seemed overkill.